desktop/libs/indexer/src/indexer/api3.py

```diff
@@ -676,3 +677,13 @@ def save_pipeline(request):
   response['message'] = request.POST.get('editorMode') == 'true' and _('Query saved successfully') or _('Notebook saved successfully')
 
   return JsonResponse(response)
+
+
+def upload(request):
+  if request.method == 'POST':
+    upload_file = request.FILES['inputfile']
+    fs = FileSystemStorage()
+    fs.save(upload_file.name, upload_file)
```

Comment:
At some point we will need to have some kind of namespace to not have filename conflicts, also to make them temporary.